web/packages/teleterm/src/ui/services/workspacesService/documentsService/connectToApp.ts

```diff
@@ -46,6 +51,19 @@ export async function connectToApp(
     return;
   }
 
+  if (isWebApp(target) && options?.launchInBrowser) {
+    captureAppLaunchInBrowser(ctx, target, telemetry);
+    // Generally, links should be opened with <a> elements.
+    // Unfortunately, in some cases it is not possible,
+    // for example, in the search bar.
+    window.open(
```

Comment:
This means that when opening an app from the search bar, the app will be opened without Connect switching to the correct workspace (like for other resources), but I think that's fine.